config.go

```diff
@@ -338,8 +338,15 @@ type Config struct {
 		// offsets. This currently requires the manual use of an OffsetManager
 		// but will eventually be automated.
 		Offsets struct {
-			// How frequently to commit updated offsets. Defaults to 1s.
-			CommitInterval time.Duration
```

Comment:
ohh shit, this was a breaking change, i.e: it breaks people using sarama-cluster with the latest version of sarama.
```
../../../../go/pkg/mod/github.com/bsm/sarama-cluster@v2.1.15+incompatible/consumer.go:452:59: c.client.config.Config.Consumer.Offsets.CommitInterval undefined (type struct { AutoCommit struct { Enable bool; Interval time.Duration }; Initial int64; Retention time.Duration; Retry struct { Max int } } has no field or method CommitInterval)
```